### DISCUSSION TITLE
WSL1 -> WSL 2 upgrade not working with 18363.1082?

### DISCUSSION THREAD

**Discussion by thsieh123** [microsoft/WSL#6186] [Q&A] [answered]
I have Win10 1909 (18363.1082). My WSL with Ubuntu 2004 seems to be ver 1. I tried to follow instruction to upgrade to WSL2 and it doesn't seem to be upgraded. There is no error message. What have I missed? 

❯ wsl --list --verbose
NAME STATE VERSION
Ubuntu-20.04 Running 1

❯ wsl --set-default-version 2
For information on key differences with WSL 2 please visit https://aka.ms/wsl2

(reboot win10)

❯ wsl --list --verbose
NAME STATE VERSION
Ubuntu-20.04 Running 1

**Comment by craigloewen-msft** [comment, marked as answer]
You're running the command `wsl --set-default-version 2` this means any NEW distribution that you install will install as a WSL 2 distro. 

Please try running `wsl --set-version Ubuntu-20.04 2` this should set that specific distro to be a version 2 distro.

Also you don't need to restart in between these commands :)

**Comment by thsieh123** [comment]
Yes! It works.
